packages/flutter/lib/src/widgets/form.dart

```diff
@@ -571,13 +618,34 @@ class FormFieldState<T> extends State<FormField<T>> with RestorationMixin {
     super.deactivate();
   }
 
+  @override
+  void initState() {
+    super.initState();
+    if (widget.forceErrorText != null) {
+      // ignore: always_specify_types
+      Future.microtask(() {
+        setState(() {
+          _errorText.value = widget.forceErrorText;
```

Comment:
Would this still work if you get rid of both the `Future.microtask` and the `setState`?

Reply:
I changed this approach entirely by marking the` _errorText` variable as `late` and setting it in initstate.